website/docs/source/v2/provisioning/ansible_local.html.md

```diff
@@ -0,0 +1,147 @@
+---
+page_title: "Ansible Local - Provisioning"
+sidebar_current: "provisioning-ansible-local"
+---
+
+# Ansible Local Provisioner
+
+**Provisioner name: `ansible_local`**
+
+The Ansible Local provisioner allows you to provision the guest using [Ansible](http://ansible.com) playbooks by executing **`ansible-playbook` directly on the guest machine**.
+
+<div class="alert alert-warn">
+  <p>
+    <strong>Warning:</strong> If you're not familiar with Ansible and Vagrant already,
+    I recommend starting with the <a href="/v2/provisioning/shell.html">shell
+    provisioner</a>. However, if you're comfortable with Vagrant already, Vagrant
+    is a great way to learn Ansible.
+  </p>
+</div>
+
+## Setup Requirements
+
+The main advantage of the Ansible Local provisioner in comparison to the [Ansible (remote) provisioner](/v2/provisioning/ansible.html) is that it does not require any additional software on your Vagrant host.
+
+On the other hand, [Ansible must obviously be installed](http://docs.ansible.com/intro_installation.html#installing-the-control-machine) on your guest machine(s).
+
+**Note:** By default, Vagrant will *try* to automatically install Ansible if it is not yet present on the guest machine (see the `install` option below for more details).
+
+## Usage
+
+This page only documents the specific parts of the `ansible_local` provisioner. General Ansible concepts like Playbook or Inventory are shortly explained in the [introduction to Ansible and Vagrant](/v2/provisioning/ansible_intro.html).
+
+The Ansible Local provisioner requires that all the Ansible Playbook files are available on the guest machine, at the location referred by the `provisioning_path` option. Usually these files are initially present on the host machine (as part of your Vagrant projet), and it is quite easy to share them with a Vagrant [Synced Folder](/v2/synced-folders/index.html).
```

Comment:
So far, I opted to not auto-create any synced folder (like Chef provisioners do). The default (`/vagrant`) should work for many projects, and it is quite simple to customize. Thoughts? Problems?